LinkSqueezer/src/GUI/MainFrame.java

```diff
@@ -326,6 +333,26 @@ public void run() {
             }
         });
     }
+    
+    // Read and Display History
+private void populateTableFromCSV() {
+    String csvFile = "D:\\Semester #6\\Software Construction\\SemesterProject\\linkSqueezer\\LinkSqueezer\\DB.csv";
```

Comment:
do not hardcode this. Give it a relative path. The DB file is the root of the project folder. Givet its relative path. Please.

Reply:
set relative path succesfully.

![Screenshot (81)](https://github.com/hussainashiqktk/linkSqueezer/assets/82904313/8290c5d1-576c-4c35-be12-b8cd1d271aa7)